dwbb/src/interface/DW_asymdata_inbuf.scala

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2024 Jiuyang Liu <liu@jiuyang.me>
+package org.chipsalliance.dwbb.interface.DW_asymdata_inbuf
+
+import chisel3._
+import chisel3.experimental.SerializableModuleParameter
+import upickle.default
+
+object Parameter {
+  implicit def rw: default.ReadWriter[Parameter] =
+    upickle.default.macroRW[Parameter]
+}
+
+case class Parameter(
+    inWidth: Int = 8,
+    outWidth: Int = 16,
+    errMode: Boolean = false,
```

Comment:
errMode...

Reply:
Using `String`s instead of `Boolean`s in the revised code.